mautrix_instagram/portal.py

```diff
@@ -388,6 +388,15 @@ async def handle_matrix_redaction(self, sender: 'u.User', event_id: EventID,
                 )
                 await self._send_delivery_receipt(redaction_event_id)
                 self.log.trace(f"Removed {reaction} after Matrix redaction")
+            return
+
+        sender.send_remote_checkpoint(
+            MessageSendCheckpointStatus.PERM_FAILURE,
+            event_id,
+            self.mxid,
+            EventType.ROOM_REDACTION,
+            error=Exception(f"No message or reaction found for redaction of {event_id}"),
+        )
```

Comment:
This should probably be in an `else` without that `return` (because the `except` block doesn't return currently)

Reply:
The `if` block that the `except` blocks are contained in return, so this should be fine.

Reply:
Oh yeah, misread that